package.json

```diff
@@ -19,7 +19,7 @@
     "lintfix:prettier": "prettier --ignore-path .gitignore --write --plugin-search-dir=. --list-different .",
     "lintfix": "tsc --noEmit && npm run lintfix:prettier && eslint --fix .",
     "prepare": "husky install && if [ -f codegen.yml ] && [ -f build-utils/generate-theme.mjs ]; then npm run setup; fi",
-    "setup": "npm run sync && npm run graphql:generate && npm run theme:generate",
+    "setup": "concurrently \"npm:sync\" \"npm:graphql:generate\" \"npm:theme:generate\"",
```

Comment:
Wouldn't `sync` need to run before `graphql:generate` (so the model schema is up to date)? Running them concurrently might not assert that.